e2e_tests/cypress/pages/connectors/Zenodo.js

```diff
@@ -0,0 +1,239 @@
+const selectors = {
+  // Navigation selectors
+  repositoriesDropdown: '#repositories-dropdown',
+  zenodoNavItem: '#nav-zenodo',
+
+  // Landing page selectors
+  breadcrumbs: 'nav[aria-label="Breadcrumb"]',
+  breadcrumbHome: '.breadcrumb .breadcrumb-item:first-child',
+  breadcrumbActive: '.breadcrumb .breadcrumb-item.active',
+  zenodoLogoContainer: 'div[data-test="zenodo-logo-container"]',
+  zenodoLogo: 'div[data-test="zenodo-logo-container"] img',
+  zenodoProjectLink: 'div[data-test="zenodo-logo-container"] a',
+
+  // Search functionality
+  searchForm: '#zenodo-search-form',
+  searchInput: 'form input[name="query"]',
+  searchSubmitButton: 'input[type="submit"]',
+  serverSchemeField: 'input[name="server_scheme"]',
+  serverPortField: 'input[name="server_port"]',
+
+  // Results table
+  resultsCard: '.card',
+  resultsHeader: '.card-header',
+  resultsTable: 'table.table-striped',
+  resultsTableBody: 'table tbody',
+  resultsTableRows: 'table tbody tr',
+  noResultsAlert: '.alert.alert-warning',
+
+  // Pagination
+  paginationNav: 'nav[aria-label="Search result pagination"]',
+  nextPageLink: 'a[title="Next page"]',
+  prevPageLink: 'a[title="Previous page"]',
```

Comment:
I would use selectors less fragile as these, since they depend on the current i18n

Reply:
fixed